src/utils/index.ts

```diff
@@ -134,14 +134,19 @@ export const isFileMessage = (message: AdminMessage | UserMessage | FileMessage)
   message && (message?.isFileMessage?.() || (message?.messageType === 'file'))
 );
 
-export const isOGMessage = (message: UserMessage): boolean => !!(
-  message && isUserMessage(message) && message?.ogMetaData && message?.ogMetaData?.url
+export const isOGMessage = (message: UserMessage | FileMessage): boolean => !!(
+  message && isUserMessage(message) && message?.ogMetaData && (
+    message.ogMetaData?.url
+    || message.ogMetaData?.title
+    || message.ogMetaData?.description
+    || message.ogMetaData?.defaultImage
```

Comment:
Q. Do We need to check all these conditions?

Reply:
Yes, we should render the OG field if one of those data exists